lib/metric_fu/templates/metrics_template.rb

```diff
@@ -23,6 +23,11 @@ def write
       write_file_data
     end
 
+    def html_filename(file)
+      file = Digest::SHA1.hexdigest(file)
+      "#{file.gsub(%r{/}, '_')}.html"
+    end
```

Comment:
I like the helper.  couldn't we just truncate it?  `"#{file.gsub(%r{/}, '_')[0..30]}".html"`